assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -194,14 +210,63 @@ const ChecklistTab = (props) => {
     [checklist]
   );
 
+  const selectElement = useCallback(
+    ({ elementId, elements, pageId }) => {
+      if (pageId) {
+        setCurrentPage({ pageId });
+      }
+      if (Array.isArray(elements)) {
+        setSelectedElementsById({
+          elementIds: elements.map(({ id }) => id),
+        });
+      } else if (elementId) {
+        setSelectedElementsById({ elementIds: [elementId] });
+      }
+    },
+    [setCurrentPage, setSelectedElementsById]
+  );
+
+  const getOnClick = useCallback(
+    ({ elementId, pageId, elements }) => {
+      if (!elementId && !pageId && !elements) {
+        return undefined;
+      }
+      return () => selectElement({ elementId, pageId, elements });
+    },
+    [selectElement]
+  );
+
+  const getHandleKeyPress = useCallback(
+    ({ elementId, elements, pageId }) => (event) => {
+      if (event.key === 'Enter') {
+        selectElement({ elementId, elements, pageId });
+      }
+    },
+    [selectElement]
+  );
+
   const renderRow = useCallback(
-    ({ message, help, id, pageGroup }) => (
-      <Row key={`guidance-${id}`} pageGroup={pageGroup}>
-        {message}
-        <HelperText>{help}</HelperText>
-      </Row>
-    ),
-    []
+    ({ message, help, id, pageGroup, ...args }) => {
+      const onClick = getOnClick(args);
+      const handleKeyPress = getHandleKeyPress(args);
+      return (
+        <Row
+          tabIndex={0}
+          onClick={onClick}
+          // note: onKeyDown does not work for "Enter"
+          onKeyUp={handleKeyPress}
+          aria-label={
+            onClick ? __('Select element for error', 'web-stories') : undefined
+          }
+          key={`guidance-${id}`}
```

Comment:
```suggestion
          key={id}
```

that should be enough I think? `id` is unique within this tree.